src/main/ruby/core/exception.rb

```diff
@@ -166,6 +166,18 @@ def to_s
 end
 
 class UncaughtThrowError < ArgumentError
+  attr_reader :tag
+  attr_reader :value
+
+  def initialize(tag, value, *rest)
+    @tag = tag
+    @value = value
+    super(*rest)
+  end
+
+  def to_s
+    sprintf(self.message, @tag)
```

Comment:
Isn't it a problem that Exception#message calls `self.to_s` and here the reverse?

Reply:
My intention was to the the `message` field, fixed now.